pkg/api/apikey.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func GetAPIKeys(c *models.ReqContext) Response {
-	query := models.GetApiKeysQuery{OrgId: c.OrgId}
+	query := models.GetApiKeysQuery{OrgId: c.OrgId, IncludeInvalid: c.QueryBool("include_invalid")}
```

Comment:
Would it be more precise if the query parameter were called "IncludeExpired"? I was unsure about what would cause an API key to be invalid.

Reply:
yes, agree with this.